index.js

```diff
@@ -120,3 +145,28 @@ exports.eejsBlock_styles = function (hook_name, args, cb) {
   args.content = args.content + eejs.require("ep_webrtc/templates/styles.html", {}, module);
   return cb();
 };
+
+function validateSettings() {
+  if(settings.ep_webrtc && settings.ep_webrtc.audio && settings.ep_webrtc.audio.disabled){
+    if (
+      settings.ep_webrtc.audio.disabled != "none" &&
+      settings.ep_webrtc.audio.disabled != "hard" &&
+      settings.ep_webrtc.audio.disabled != "soft"
+    ) {
+      throw Error("Invalid value in settings for ep_webrtc.audio.disabled")
+    }
+  }
+
+  if(settings.ep_webrtc && settings.ep_webrtc.video && settings.ep_webrtc.video.disabled){
+    if (
+      settings.ep_webrtc.video.disabled != "none" &&
+      settings.ep_webrtc.video.disabled != "hard" &&
+      settings.ep_webrtc.video.disabled != "soft"
+    ) {
+      throw Error("Invalid value in settings for ep_webrtc.video.disabled")
+    }
+  }
+}
+
+// Validate settings.json now so that the admin notices any errors right away
+validateSettings()
```

Comment:
This brings up error messages during startup but it doesn't stop the server from loading. Though I suppose it does stop the plugin from loading. Is there a more proper way to handle this?